src/app/features/public-form/edit-prefilled-values/edit-prefilled-values.component.html

```diff
@@ -0,0 +1,60 @@
+<p i18n class="component-info">
+  Set the default value related to the field. After setting the default value,
+  the selected value will be automatically pre-filled in the corresponding field
+  of the public form. This will apply only to this specific public form and will
+  not impact other forms or configurations.
+</p>
+<div class="defaultvalue-container" [class.disabled]="formControl.disabled">
+  <form [formGroup]="prefilledValueSettings">
+    <div formArrayName="prefilledvalue">
+      <div
+        *ngFor="let fieldGroup of prefilledValues.controls; let i = index"
+        [formGroupName]="i"
+      >
+        <div class="flex-row justify-space-between align-center container">
```

Comment:
The fields have line-breaks very early, even on a fairly wide screen. I think the form field could reduce its width a bit more before switching the layout. (Maybe there is an unnecessarily large min-width on the entity-field-select?)
![image](https://github.com/user-attachments/assets/d59cbeae-52db-4c21-9d44-d0b10f3b157f)